remotedb/remotedb_test.go

```diff
@@ -1,5 +1,3 @@
-// +build remotedb
```

Comment:
In the `remotedb` case, there is no need `//go:build xxx` comment like `//go:build rocksdb`.

Reply:
Basically, there is nothing on originally `tm-db`. And I don't know why we put it.

- original tm-db: https://github.com/tendermint/tm-db/blob/v0.6.6/remotedb/remotedb_test.go
- our commit: https://github.com/line/tm-db/pull/5/files#diff-4bb259d002fe73c2d2e9b8a2e8f0bf841c62093528aed2f2bde3520b4a09bbf7R1

Reply:
`remote_db_test.go` file need `remotedb` tag for compile since has `// +build remotedb` option.
So, this file is executed when execute `make test-remotedb` or `make bench-remotedb`.
I think it is need when you really want to execute this file.

Please, check it again.

Reply:
I've already checked it. The `remotedb` tag only exists in this `remotedb_test.go`. Please check the `remotedb.go` and `remotedb` packages.
- https://github.com/line/tm-db/blob/main/remotedb/remotedb.go
- https://github.com/line/tm-db/tree/main/remotedb

And when we put the `remotedb` tag, it doesn't exist anywhere else.
- https://github.com/line/tm-db/blob/a0858fa3669a74666a8c675859c0124336591667/remotedb/db_test.go
- https://github.com/line/tm-db/blob/a0858fa3669a74666a8c675859c0124336591667/remotedb/db.go
- https://github.com/line/tm-db/tree/a0858fa3669a74666a8c675859c0124336591667/remotedb

Of course, It can say `tm-db-v0.6.4` is the same.
- https://github.com/line/tm-db/tree/fcfa67dd3c70210cf6d852a0cb8ba197e4f3cf50/remotedb

So, I don't know why we put it on `remotedb_test.go` only. Do you know that? Why do we separate into default build and `remotedb` build on `make test-remotedb/make bench-remotedb`? I just think it can only test/bench without `remotedb_test.go` by default. And I've not understood to execute only this file. If you know, please teach me.

In addition, I removed tasks of `test-x/bench-x` related to `memdb/goleveldb/prefixdb` because its build tags are nothing on source codes.

Thanks.

Reply:
I think this tags are added for testing and doing benchmark each db in previous version. If you think this test and benchmarking is not need any more in current version, I think it is also a good idea to remove it.

Reply:
> I think this tags are added for testing and doing benchmark each db in previous version.

Sorry, I'm not sure what you mean. First, `remotedb_test .go` doesn't have the `BenchXxx` function. If we benchmark with `remotedb` via `make bench-remotedb`, it does nothing. And I've not understood yet about separately testing/benchmarking. About benchmarking, if we want to benchmark each db, it can help you take a look separately. Does it need for us even if `memdb/gorockdb` doesn't exist? I mean we can't test/bench each dbs now. And in testing, I think it cannot test `remotedb_test.go` by default. I think it's bad because `remotedb.go` isn't tested by default even though we can use `remotedb` without the `remotedb` tag.

In my opinion, If I only mention the `remotedb` build tag, we should give it to the whole package. And we should make `BenchXxx` in `remotedb_test.go`.

Reply:
In the previous version, each db was existed in the each directory. But current version joined in the one root path of this project.  So I thought the ci or directly someone maybe tested or did benchmark each db. So, I guess these tag was added in order to keep this rule. 

But If you don't need any more this test or benchmark, I think it is good to remove it.

Reply:
I understand your thoughts. Thanks for your help.